defaults/main.yml

```diff
@@ -3,16 +3,18 @@
 # Enable or disable PKI support
 pki: True
 
+# Default domain
+pki_domain: '{{ ansible_domain }}'
```

Comment:
In case a domain is not defined, this should probably be something like:
```
pki_domain: '{{ ansible_domain|d("") }}'
```
I'll need to check if the new code works on a host without an `ansible_domain` set.